test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -2770,24 +2770,6 @@ BOOST_AUTO_TEST_CASE(virtual_function_usage_in_constructor_arguments)
 	ABI_CHECK(callContractFunction("getA()"), encodeArgs(2));
 }
 
-BOOST_AUTO_TEST_CASE(constructor_argument_overriding)
```

Comment:
This was the test for the semantics of the feature we removed (overriding constructor arguments by giving them multiple times).